subject-similarity-groups/src/main/scala/edu/gatech/cse6250/main/Main.scala

```diff
@@ -42,7 +44,8 @@ object Main {
     val sqlContext = spark.sqlContext
     import org.apache.spark.sql.functions._
 
-    var base = ""
+    var base = "file:///mnt/host/c/Users/Benjamin/Documents/GaTech/sleep-predictions-through-deep-learning/subject-similarity-groups/";
```

Comment:
Will this break things? This is just something that needs to be updated per-machine right now?

Reply:
ya - upon submitting I would not have my personal path. But I kept in there as an example in case someone runs it on a local machine.

Reply:
I propose making a separate task once all our code is in to handle physical machine pathing.